pydm/widgets/timeplot_curve_editor.py

```diff
@@ -0,0 +1,71 @@
+from ..PyQt.QtGui import QDialog, QVBoxLayout, QHBoxLayout, QTableView, QAbstractItemView, QSpacerItem, QSizePolicy, QDialogButtonBox, QPushButton, QItemSelection
+from ..PyQt.QtCore import Qt, pyqtSlot
+from ..PyQt.QtDesigner import QDesignerFormWindowInterface
+from timeplot_table_model import PyDMTimePlotCurvesModel
```

Comment:
I only got this line working by relative import (adding the dot):

`from timeplot_table_model import PyDMTimePlotCurvesModel`